website/project/views/discussions.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+from website.project.decorators import (
+    must_be_valid_project,
+    must_have_permission,
+    must_not_be_registration,
+)
+from website.util.permissions import ADMIN
+
+from website.project.mailing_list import route_message
+
+
+###############################################################################
+# View Functions
+###############################################################################
+
+@must_be_valid_project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def get_node_discussions(node, auth, **kwargs):
```

Comment:
We talked in person about `discussion` vs. `mailing_list` for nomenclature. The previous PR has this comment:

> Currently, I have very inconsistently used the terms "Project Mailing" and "Email Discussions" throughout both the front and back ends of this feature, largely due to the issue mentioned below about future integrations vs. current clarity.
>
The use of both of these is fine, but a more consistent use pattern needs to be used for it; for example, "Project Mailing" could be used for functionality specific to this feature while "Discussions" could be used for things that will likely be used in the future by other parts of the larger discussions forums feature.
>
Alternatively, all mention of "Discussions" could be removed and simply re-applied whenever more features are added that want to integrate with this one.

I think we should go with that last option, renaming `discussion` to `mailing_list`.